packages/core-js/test/utils/hardhat/rpc.test.js

```diff
@@ -1,13 +1,25 @@
 const assert = require('assert/strict');
 const sinon = require('sinon');
-const { takeSnapshot, restoreSnapshot, fastForward } = require('../../../utils/hardhat/rpc');
+const {
+  takeSnapshot,
+  restoreSnapshot,
+  fastForward,
+  fastForwardTo,
+  getTime,
+} = require('../../../utils/hardhat/rpc');
 
 const fakeProvider = {
   send() {
     return new Promise((resolve) => {
       resolve(42);
     });
   },
```

Comment:
The same as the other suggestion:
```suggestion
  async send() {
    return 42;
  },
```

Reply:
Implemented

Reply:
Not seeing it implemeted on the diff 🤔 https://github.com/Synthetixio/synthetix-v3/blob/6fd278a159ece3b2067328486556adc5c3326d06/packages/core-js/test/utils/hardhat/rpc.test.js